src/isar/apis/api.py

```diff
@@ -93,6 +93,7 @@ def _create_scheduler_router(self) -> APIRouter:
             "/schedule/start-mission/{id}",
             self.scheduling_controller.start_mission_by_id,
             methods=["POST"],
+            deprecated=True,
```

Comment:
@tsundvoll @aestene What do you guys think about this? End point still usable, but If I understand correct we will not use this for other then testing locally?

Reply:
I think it works well until we decide to remove it.